src/OpenTelemetry.Exporter.Geneva/ExceptionStackExportOptions.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="ExceptionStackExportOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Exporter.Geneva;
+
+public enum ExceptionStackExportOptions
```

Comment:
I would rename this again! Sorry 🤣 

Two reasons...

* [Naming conventions for enums says they should be plural only when using `[Flags]`](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations)

* Anything called "Options" should be a POCO, IMO. "Options" classes being special things that are accessed through Options API and bindable to IConfiguration.

`ExceptionStackExportType` or `ExceptionStackExportMode` maybe?